docs/resources-overview.md

```diff
@@ -32,13 +33,13 @@ To see only objects of a specific type, for example to see the webhook and
 controller deployments inside Knative Serving, you can run
 `kubectl -n knative-serving get deployments`.
 
-The Knative Serving controller creates Kubernetes and Istio
-resources when Knative Serving resources are created and updated. It will also
-create Build resources when provided in the Configuration spec. These
-sub-resources will be created in the same namespace as their parent Knative
-Serving resource, _not_ the `knative-serving` namespace. For example, if you
-create a Knative Serivce in namespace 'foo' the cooresponding Istio resources
-will also be in namespace 'foo'.
+The Knative Serving controller creates Kubernetes and Istio resources when
+Knative Serving resources are created and updated. It will also create Build
+resources when provided in the Configuration spec. These sub-resources will be
+created in the same namespace as their parent Knative Serving resource, _not_
+the `knative-serving` namespace. For example, if you create a Knative Serivce in
+namespace 'foo' the cooresponding Istio resources will also be in namespace
```

Comment:
<!--typo-bot-->
```suggestion
namespace 'foo' the corresponding Istio resources will also be in namespace
```

Reply:
This will be fixed by the bot in another PR, then the next run will fix the formatting.